diff --git a/node_modules/react-native-stopwatch-timer/lib/stopwatch.js b/node_modules/react-native-stopwatch-timer/lib/stopwatch.js
index c59be85..e2b972a 100644
--- a/node_modules/react-native-stopwatch-timer/lib/stopwatch.js
+++ b/node_modules/react-native-stopwatch-timer/lib/stopwatch.js
@@ -51,17 +51,20 @@ class StopWatch extends Component {
       this.start();
     }
   }
-
-  componentWillReceiveProps(newProps) {
-    if(newProps.start) {
-      this.start();
-    } else {
-      this.stop();
+  componentDidUpdate(prevProps) {
+    if (this.props.start !== prevProps.start) {
+      if(this.props.start) {
+          this.start();
+      } else {
+          this.stop();
+      }
     }
-    if(newProps.reset) {
-      this.reset();
+    if (this.props.reset !== prevProps.reset) {
+      if(this.props.reset) {
+          this.reset(this.props.totalDuration);
+      }
     }
-  }
+}
 
   componentWillUnmount() {
      clearInterval(this.interval);
